batctl/Makefile

```diff
@@ -1,54 +1,47 @@
-#
-# Copyright (C) 2014 OpenWrt.org
-#
-# This is free software, licensed under the GNU General Public License v2.
-# See /LICENSE for more information.
-#
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
Do we need this if PKG_LICENSE is defined below anyway?

Reply:
I understand the PKG_LICENSE as too much reduced debian/copyright. But this information here is about the Makefile

Reply:
Okay, I was just asking here, do as you prefer.